### PR TITLE
Documentation fixes. 

### PR DESCRIPTION
**Pull request by sbhaktha**
* Correct command-line to run `createIndex` 
* Add steps regarding Postgres Sql install requirement.

@aimichal : simple!
@carissas : fyi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ IKE (Interactive Knowledge Extraction)
 [Live Demo](http://ike.allenai.org/)
 
 ## Run Locally
-1. Run `sbt`
-2. Enter the `reStart` command
-3. Open http://localhost:8080 in a browser
+1. Install Postgres SQL locally and create a database for use by IKE. IKE needs this to store the tables you create.
```
Comment (aimichal) [inline README.md]: Technically it's called "PostgreSQL" :-)

**Comment by aimichal** [comment]
looks good to me.

File: README.md (hunk 1 above)
Comment (sbhaktha) [inline README.md]: Yeah I tried `PostgresSql` and `Postgres Sql` -- neither looked right, thanks  :)

**Comment by sbhaktha** [comment]
done. merging!

**Pull request merged**